web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml

```diff
@@ -22,8 +26,34 @@
     <link rel="manifest" href="/assets/manifest.json">
     <link rel="stylesheet" href="/css/front-end.css"/>
     <link rel="stylesheet" href="/css/main.css" asp-append-version="true"/>
+    @if (!string.IsNullOrEmpty(connString))
+    {
+        <script type="text/javascript" src="https://js.monitor.azure.com/scripts/b/ext/ai.clck.2.min.js"></script>
+        <script type="text/javascript">
+ 		    var clickPluginInstance = new Microsoft.ApplicationInsights.ClickAnalyticsPlugin();
```

Comment:
[nitpick] Tab spacing looks a little off

Reply:
Will have a look. Annoyingly Rider refuses to reformat it.